addons/website/static/src/scss/website.scss

```diff
@@ -1341,16 +1341,19 @@ header {
 }
 
 #wrapwrap.o_header_overlay {
-    > header:not(.o_header_affixed):not(.o_top_menu_collapse_shown):not(.o_header_sidebar) {
+    > header:not(.o_header_affixed):not(.o_header_sidebar) {
         @include o-position-absolute(0, 0, auto, 0);
         z-index: 1000;
 
         > .navbar {
-            @include o-apply-colors(1); // Reset to default colored components
-            // Prevent the nav color/gradient to impact the header color.
             background-color: transparent !important;
             background-image: none !important;
             border-color: transparent;
+        }
+
+        &:not(.o_top_menu_collapse_shown) > .navbar {
+            @include o-apply-colors(1); // Reset to default colored components
+            // Prevent the nav color/gradient to impact the header color.
             color: inherit;
```

Comment:
It's already the case in stable, but the fact that we set the color under `:not(.o_top_menu_collapse_shown)` means that upon opening the menu, other elements on the navbar lose their custom color.
I think a solution is to set `color: inherit !important;` on the previous selector (the one without `:not(.o_top_menu_collapse_shown)`).

Reply:
Thanks :+1:  I made the changes and it seems to work well.